package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gpii-pouch",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "node src/tests/all-tests.js"
+  },
+  "dependencies": {
+    "express":         "4.10.5",
+    "express-pouchdb": "0.14.0",
+    "infusion":        "git://github.com/fluid-project/infusion.git#df5f8cfabf815a4086b778e73125c3c952dda4ec",
+    "memdown":         "1.0.0",
+    "pouchdb":         "3.0.5",
```

Comment:
Hi, @amb26.  I reviewed their versions, the change was introduced in 3.2.1.  Just to confirm, are you suggesting updating to make the build work on Windows, or suggesting keeping something pre-3.2.1 to avoid problems you encountered?  Please confirm.